samples/nrf9160/aws_fota/src/main.c

```diff
@@ -0,0 +1,566 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <uart.h>
+#include <string.h>
+
+#include <net/mqtt.h>
+#include <net/socket.h>
+#include <net/aws_fota.h>
+#include <lte_lc.h>
+#if defined(CONFIG_BSD_LIBRARY)
+
+#include "nrf_inbuilt_key.h"
+#endif
+
+#define CONFIG_NRF_CLOUD_SEC_TAG 16842753
```

Comment:
You should not define `CONFIG_*` symbols on your own, especially when they cover existing Kconfig options, it's very confusing. Also, `NRF_CLOUD` does not seem like a proper name for this sample.